_posts/samus_returns/2024-06-11-export-exheader.md

```diff
@@ -3,9 +3,11 @@ layout: post
 title: "How to extract your Metroid: Samus Returns ExHeader"
 tags: samus_returns
 ---
-You can either extract your ExHeader using Godmode9 from your 3DS, or by using [ctrtool](https://github.com/3DSGuy/Project_CTR/releases/tag/ctrtool-v1.2.0) or [pyctr](https://github.com/ihaveamac/pyctr) on your computer. We'll show you how to do it using Godmode9.
+You can either extract your ExHeader using Godmode9 from your 3DS, or by using [CTRTool](https://github.com/3DSGuy/Project_CTR/releases/tag/ctrtool-v1.2.0) or [pyctr](https://github.com/ihaveamac/pyctr) on your computer. We'll show you how to do it using Godmode9 and CTRTool.
```

Comment:
Why pyctr? what does that one do differently/better than ctrtool? The API is unstable, and you dont give any other mention here, so this feels a bit uselees.

Reply:
Yeah I only had it since it was mentioned like once, but I'll just remove it.